bob.js

```diff
@@ -807,16 +808,25 @@ meCmd = (message) => {
     stats = client.getUserStatsForGame.all(message.author.id, game);
     if (stats.length > 0) {
         meString = "**" + game + "**\n";
+        var maxNumberLengths = [1, 1, 1, 1, 1, 1];
         stats.forEach((line) => {
-            meString += "  " + line.category + "\n "
-                    + "   :checkered_flag: " + line.races
-                    + "   :first_place: " + line.gold
-                    + "   :second_place: " + line.silver
-                    + "   :third_place: " + line.bronze
-                    + "   :x: " + line.ffs
-                    + "   " + emotes.ppjSmug + " " + Math.floor(line.elo)
-                    + "   :stopwatch: " + (line.pb > 0 ? formatTime(line.pb) : "--:--:--")
-                    + "\n";
+            maxNumberLengths[0] = Math.max(maxNumberLengths[0], line.races.toString().length);
+            maxNumberLengths[1] = Math.max(maxNumberLengths[1], line.gold.toString().length);
+            maxNumberLengths[2] = Math.max(maxNumberLengths[2], line.silver.toString().length);
+            maxNumberLengths[3] = Math.max(maxNumberLengths[3], line.bronze.toString().length);
+            maxNumberLengths[4] = Math.max(maxNumberLengths[4], line.ffs.toString().length);
+            maxNumberLengths[5] = Math.max(maxNumberLengths[5], Math.floor(line.elo).toString().length);
+        });
+        stats.forEach((line) => {
+            meString += "  " + line.category
+                    + "\n    :checkered_flag: `" + addSpaces(line.races, maxNumberLengths[0])
+                    + "`   :first_place: `" + addSpaces(line.gold, maxNumberLengths[1])
+                    + "`   :second_place: `" + addSpaces(line.silver, maxNumberLengths[2])
+                    + "`   :third_place: `" + addSpaces(line.bronze, maxNumberLengths[3])
+                    + "`   :x: `" + addSpaces(line.ffs, maxNumberLengths[4])
+                    + "`   " + emotes.ppjSmug + " `" + addSpaces(Math.floor(line.elo), maxNumberLengths[5])
+                    + "`   :stopwatch: `" + (line.pb > 0 ? addSpaces(formatTime(line.pb), 11) : "--:--:--.--")
```

Comment:
We should keep track of a `maxNumberLengths` here too, right? If someone only has short sub hour PBs like die% or styrofoam% or something, the times will have a lot of extra spaces before them since the length of 11 is hardcoded. (That, or just don't cut off the initial 0s, which I'm kind of leaning towards.)